docpages/INDEX.md

```diff
@@ -35,12 +35,10 @@ The library may work fine in other operating systems too, if you run a D++ bot o
 
 ## Getting started
 * [GitHub Repository](https://github.com/brainboxdotcc/DPP)
-* [Discord Server](https://discord.gg/dpp)
```

Comment:
I like the PR but I don't think this change below is needed?
it just shortens the links which harms SEO